assets/src/scss/new-identity/style.scss

```diff
@@ -52,6 +52,22 @@
   --block-spreadsheet-blue-header--background: #167f82;
   --block-spreadsheet-blue-odd-row--background: var(--beige-200);
   --block-spreadsheet-blue-even-row--background: var(--beige-100);
+  --table-header-grey--background: var(--grey-800);
+  --table-odd-row-grey--background: var(--grey-200);
+  --table-even-row-grey--background: var(--grey-100);
+  --table-footer-grey--background: var(--grey-200);
+  --table-header-green--background: var(--p4-dark-green-800);
+  --table-odd-row-green--background: var(--beige-200);
+  --table-even-row-green--background: var(--beige-100);
+  --table-footer-green--background: var(--beige-200);
+  --table-header-blue--background: #167f82;
+  --table-odd-row-blue--background: var(--beige-200);
+  --table-even-row-blue--background: var(--beige-100);
+  --table-footer-blue--background: var(--beige-200);
+  --table-header-gp-green--background: var(--gp-green-800);
```

Comment:
It's more a matter of preference, but I would personally prefer to use `dark-green` rather than `gp-green` in the name, I feel it's more clear what the difference is

Reply:
Actually we already have a _Dark green_ for the new identity styles, so this was more to reference the color itself so as not to cause confusion when looking at the code.

Reply:
Well it's `green` or `dark green` right? Also for the colors' names I would use `Green` and `Dark Green` (like for the Spreadsheet block), I think for editors it's clearer... But maybe that should be discussed with Houssam, maybe he would prefer to use the names that correspond to the color palette instead 🤷‍♀️

Reply:
I guess that's for him to decide, which labels he prefers, but either way we should make them consistent with the Spreadsheet ones 🙂